infrastructure/proto/log_replication_metadata.proto

```diff
@@ -130,6 +130,7 @@ message ReplicationEvent {
   enum ReplicationEventType {
     FORCE_SNAPSHOT_SYNC = 0;
     UPGRADE_COMPLETION_FORCE_SNAPSHOT_SYNC = 1; // Intent to queue force sync for all sessions
+    UPGRADE_COMPLETION_FORCE_SNAPSHOT_SYNC_RETRIGGER = 2; // If initial event is aborted this is used as placeholder so a second event is not queued
```

Comment:
we do not require another event type

Reply:
Whenever we update the uuid since we currently do not differentiate between put/update operations if we were to use the same event type then it would again input another discovery service event for a force sync for the one that had just been re-triggered.